### ISSUE TITLE
Generation fails when using a game path that isn't the same folder as the Steam installation

### ISSUE THREAD

**Issue by Tourmi** [Lakifume/True-Randomization#39]
DLC fails to be detected, and generation ends up failing due to this.

Fixing this shouldn't be too bad, as the Steam library folder has a `libraryfolder.vdf` file at its root that points to the Steam executable. 

So before going into Steam/config/loginusers.vdf, we should first check that we aren't in a different library, by checking if the {Steam}/libraryfolder.vdf file exists. If it exists, open the vdf file, check the `libraryfolder.launcher` value, and adjust the steam path based on that

libraryfolder.vdf file structure
```vdf
"libraryfolder"
{
	"contentid"		"6614107190113200476"
	"label"		""
	"launcher"		"C:\\Program Files (x86)\\Steam\\steam.exe"
}

```

- libraryfolder.vdf path : `T:\SteamLibrary\libraryfolder.vdf` 
  - Note: the `SteamLibrary` folder can have any name, so we can only go up the hierarchy using parent accessors (`../../..`) or using the `steamapps` part of the game install path
- Bloodstained install path: `T:\SteamLibrary\steamapps\common\Bloodstained Ritual of the Night`
- Steam install path: `C:\Program Files (x86)\Steam`
- `loginusers.vdf`: `C:\Program Files (x86)\Steam\config\loginusers.vdf`
- The `localconfig.vdf` file is at `C:\Program Files (x86)\Steam\userdata\userid\config\localconfig.vdf`

**Comment by Tourmi** [comment]
Potentially fixed: 461f15d567b0243bde945b71547adbcd8c1cfff0

**Comment by Tourmi** [comment]
Solution works!
